core/css.js

```diff
@@ -516,6 +513,10 @@ Blockly.Css.CONTENT = [
     'overflow-x: visible;',
     'overflow-y: auto;',
     'position: absolute;',
+    'user-select: none;',
+    '-moz-user-select: none;',
+    '-ms-user-select: none;',
+    '-webkit-user-select: none;',
```

Comment:
Double-clicking on a category really shouldn't select the text.  Categories are basically buttons.